docs/src/main/sphinx/release/release-478.md

```diff
@@ -0,0 +1,80 @@
+# Release 478 (22 Oct 2025)
+
+## General
+
+* Add support for column lineage in `UNNEST` clauses. ({issue}`16946`)
+* Add `retry-policy.allowed` configuration property to specify which query retry
+  policies can be selected by the user. ({issue}`26628`)
+* Add support for loading plugins from multiple directories. ({issue}`26855`)
+* Add the `/v1/integrations/gateway` endpoint that exposes additional cluster metrics for Trino Gateway.
+  ({issue}`26548`)
+* Allow dropping an uninitialized catalog that failed to load. ({issue}`26918`)
+* Improve performance of queries with an `ORDER BY` clause using `varchar` or `varbinary` types. ({issue}`26725`)
+* Improve performance of `MERGE` statements involving a `NOT MATCHED` case. ({issue}`26759`)
+* Improve performance of queries with joins which spill to disk. ({issue}`26076`)
+* Fix potential incorrect results when reading `row` type. ({issue}`26806`)
+* Return all catalogs, including uninitialized ones, for queries from `metadata.catalogs`. ({issue}`26918`)
+* Fix `EXPLAIN ANALYZE` planning so that it executes with the same plan as would be used to execute the query
+  being analyzed. ({issue}`26938`)
+* Fix row pattern matching logical navigations in running semantics to be always constraint to current match.
+  Previously, a logical navigation function such as `FIRST` could return position outside the current match.
+  ({issue}`26981`)
+* Return correct "Table does not exist" error when querying a non-existent table in an existing schema within
+  `system` catalog. ({issue}`27047`)
+* Avoid logging an exception when listing columns from tables from a set of catalogs that includes the `system`
+  catalog. ({issue}`27045`)
+
+## Security
+
+* Propagate `queryId` to the [Open Policy Agent](/security/opa-access-control)
+  authorizer. ({issue}`26851`)
+
+## Docker image
+
+* Run Trino on JDK 25.0.0 (build 36). ({issue}`26693`)
+
+## Delta Lake connector
+
+* Fix failure when reading `map` type with `json` value type when a value is `NULL`. ({issue}`26700`)
```

Comment:
This is still not clear to me. Does "map type" refer to Delta Lake's type? Does "json value type" refer to Trino's type?

Reply:
The map type also refers to Trino type in my understanding. The PR contains a below test:  
```sql
MAP(ARRAY['key1'], ARRAY[NULL])
```
The column type is `map(varchar, json)`.